### ISSUE TITLE
Trenton Lighthammer Blacksmithing Quest Bug

### ISSUE THREAD

**Issue by Bootscoot** [elysium-project/server#827]
I turned in 2 of the 3 quests "The World At Your Feet" and "The Mithril Kid" following "The Mithril Order" and the quest "Did you lose this?" appeared. I completed that quest and the 3rd blacksmith turn in quest, "A Good Head On Your Shoulders" disappeared and was not obtainable or completable. I have no other way to obtain the Ornate Mithril Helm Recipe and it will make the Ironforge Armorsmith quest difficult to complete.

**Comment by megadus666** [comment]
Yeah, totally, i got the same thing on Darrowshire. the quest for ornate mithril helm dissappeared.

**Comment by megadus666** [comment]
Can someone fix it pls?:) the issue is probably that the "Did you lose this?" quest deletes all the smithing quests for some reason, but then it should only appear after completing all 3 of the smithing quests, not just 2/3 of 'em. Thx
